conjure_oxide/src/main.rs

```diff
@@ -299,6 +305,50 @@ pub fn main() -> AnyhowResult<()> {
 
 /// Runs the solver
 fn run_solver(cli: &Cli, model: Model) -> anyhow::Result<()> {
+    let solver = cli.solver;
+    match solver {
+        Some(sol_family) => match sol_family {
+            SolverFamily::SAT => run_sat_solver(cli, model),
+            SolverFamily::Minion => run_minion(cli, model),
+        },
+        None => run_sat_solver(cli, model),
```

Comment:
give an error instead of defaulting to SAT

Reply:
you can even call `bug` since this should not be reachable, as there is always a default.